docusaurus.config.js

```diff
@@ -38,81 +39,89 @@ module.exports = {
     algolia: {
       apiKey: "f5dfbee43cfa4c5024b10045c6d91461",
       indexName: "demisto",
-      algoliaOptions: { typoTolerance: false, hitsPerPage: 1000, filters: 'type:lvl1 OR type:content' } // Optional, if provided by Algolia
+      algoliaOptions: {
```

Comment:
im not big fan of changing things just for prettifying as i prefer to preserve the git history

Reply:
I agree, although the prettified result is much easier to diff than the original. I'll work on reverting all formatting changes to `docusaurus.config.js`.